streaming/streaming.go

```diff
@@ -445,16 +445,22 @@ func LaunchDataflowJob(ctx context.Context, targetProfile profiles.TargetProfile
 	} else {
 		dataflowHostProjectId = dataflowCfg.HostProjectId
 	}
+
 	dataflowSubnetwork := ""
+
+	// if custom network is not selected, use public IP. Typical internal testing flow
+	workerIpAddressConfig := dataflowpb.WorkerIPAddressConfiguration_WORKER_IP_PUBLIC
+
 	if dataflowCfg.Network != "" {
```

Comment:
and also != nil?

Reply:
This was existing check, tested it with and without specifying VPC. So seems fine as is.